src/latexify/frontend.py

```diff
@@ -55,6 +59,10 @@ def get_latex(
         tree = transformers.IdentifierReplacer(identifiers).visit(tree)
     if reduce_assignments:
         tree = transformers.AssignmentReducer().visit(tree)
+    # assume that both prefixes and PREFIXES are sets.
+    # merge and de-duplicate the prefix list.
+    merged_prefixes = set(constants.PREFIXES) | prefixes
```

Comment:
Since PREFIXES is no longer referenced from somewhere else, the definition can be converted to a set.